jib-gradle-plugin/build.gradle

```diff
@@ -70,6 +70,7 @@ dependencies {
   testCompile 'junit:junit:4.12'
   testCompile 'org.mockito:mockito-core:2.23.4'
   testCompile 'org.slf4j:slf4j-api:1.7.25'
+  testCompile 'org.springframework.boot:spring-boot-gradle-plugin:2.1.6.RELEASE'
```

Comment:
Let's add a comment

`// only for testing a concrete Spring Boot example in a test (not for test infrastructure)`

Reply:
You have write access to this branch of my fork of jib. So you can do such small changes directly.